contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -157,6 +276,20 @@ def init_rename_column_dict(columns_array: pd.core.indexes.base.Index) -> dict:
     return columns_dict
 
 
+def strip_unwanted_columns(df: pd.DataFrame) -> pd.DataFrame:
+    columns_to_strip = []
+    for column_name in UNWANTED_DATA_TO_UPLOAD:
+        if column_name in df.columns:
+            columns_to_strip.append(column_name)
+
+    if columns_to_strip:
+        stripped_df = df.drop(columns=columns_to_strip)
+    else:
+        stripped_df = df
+
+    return stripped_df
```

Comment:
```suggestion
    if not columns_to_strip:
        return df
    return df.drop(columns=columns_to_strip)
```

It feels wrong to assign `df` to `stripped_df` when there are no columns to strip, because `df` is, well, not stripped. But this is subjective, I think.